admin_guide/overcommit.adoc

```diff
@@ -42,6 +42,59 @@ the specified resource as is available with the lowest local priority. In low
 resource situations, containers that specify no resource requests are given the
 lowest quality of service.
 
+[[tune-buffer-chunk-limit]]
+=== Tune Buffer Chunk Limit
+
+If Fluentd logger is unable to keep up with a high number of logs, you will need
+to increase the compute resource values. The memory limit is used to calculate
+the Fluentd `buffer_queue_limit` by dividing the resource memory limit by the
+number of output multiplied by the `buffer_chunk_size`. The default
```

Comment:
Documentations should be changed based on https://bugzilla.redhat.com/show_bug.cgi?id=1466005#c5